roundware/lib/api.py

```diff
@@ -444,6 +444,12 @@ def check_for_single_audiotrack(session_id):
         ret = True
     return ret
 
+def check_is_active(session_id):
+    session = models.Session.objects.select_related('project').get(id=session_id)
```

Comment:
Usually, I would wrap every .get() request in a try/catch block. Since this function is called in a try/catch block, the exception will be caught there and a 400 will be returned.